tagger.go

```diff
@@ -68,22 +78,36 @@ func checkLinodeTagsAgainstConfig(linodes []linodego.Instance, rules []TagRule)
 
 				// check `absent` tags to remove unwanted tags
 				for _, tag := range linode.Tags {
+					report[tag] = map[string]string{}
 					if !slices.Contains(rule.Tags.Absent, tag) {
 						// if this tag is not on the `absent` list,
 						// we can persist it through to the new tag set
 						newTags = append(newTags, tag)
+					} else {
+						// if this tag *is* on the `absent` list, we'll be removing it later
+						absentReportCounter += 1
+						report[tag]["absent"] = strconv.Itoa(absentReportCounter)
```

Comment:
Is there a reason we're storing this counter as a string in the report map? Can it remain an int in the map? 🤔 
```suggestion
						report[tag]["absent"]++
```